### ISSUE TITLE
Travis build fails if Prometheus pod is not already running

### ISSUE THREAD

**Issue by jheretic** [m-lab/prometheus-support#310]
Not sure if this should be a bug or an enhancement, but anyway:

The Travis deployment process for prometheus-federation on sandbox is somewhat fragile for testing larger changes. Particularly, if there's a change to Prometheus being tested that may result in Prometheus failing to start correctly, or crash, one can't simply revert it by running a known-good build on Travis. https://github.com/m-lab/prometheus-support/blob/8f3583bab5ff46e208c5989cd660f68b793295ca/apply-global-prometheus.sh#L239 copies target files directly to the running Prometheus pod as one of the last steps of the deployment script. But if a previous test left Prometheus in a bad state, this fails and the Travis build process errors out. The Prometheus pod must be fixed manually in order for the build to succeed. It would be ideal to be able to rapidly revert to a known good state after testing a potentially breaking change.

**Comment by stephen-soltesz** [comment]
I agree. And, I think I've encountered this bug before too. I think it would be acceptable to make file copy errors non-fatal, and just print a big scary message. Those files don't change often and being able to automatically recover the deployment is a higher priority.
